ocaml/lambda/translcore.ml

```diff
@@ -1098,7 +1095,7 @@ and transl_apply ~scopes
         }
   in
   let rec build_apply lam args loc pos ap_mode = function
-    | Omitted { mode_closure; mode_arg; mode_ret; ty_arg; ty_env } :: l ->
```

Comment:
Is there any particular reason for removing the type from here?

Reply:
> anything that is stored in a block is a value

Except for e.g. all-float records, right?

Reply:
> >anything that is stored in a block is a value
> 
> Except for e.g. all-float records, right?

I don't see the remark that Mark is replying to here.  Maybe something odd is happening in the github interace?